data/en/dotnettocftype.json

```diff
@@ -1,19 +1,23 @@
 {
 	"name":"dotnetToCFType",
 	"type":"function",
-	"syntax":"dotnetToCFType(variable_name)",
+	"syntax":"dotnetToCFType(variableName)",
 	"returns":"any",
 	"related":[],
-	"description":" Explicitly converts a value returned by a .NET method to the corresponding ColdFusion data type.",
+	"description":"Explicitly converts a value returned by a .NET method to the corresponding ColdFusion data type.",
 	"params": [
-		{"name":"variable_name","description":"","required":true,"default":"","type":"Varible","values":[]}
+		{"name":"variableName","description":"Name of the .NET variable to convert.","required":true,"default":"","type":"Varible","values":[]}
 
 	],
 	"engines": {
-		"coldfusion": {"minimum_version":"", "notes":"", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-c-d/dotnettocftype.html"}
+		"coldfusion": {"minimum_version":"8", "notes":"CF2018 changed `variable_name` to `variableName`.", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-c-d/dotnettocftype.html"}
```

Comment:
Named parameters were introduced in CF2018 so I'm not sure what it means to have the name changed.

Reply:
Changes made.  CF2018 comment removed.